process/transaction/shardProcess.go

```diff
@@ -503,6 +509,78 @@ func (txProc *txProcessor) processBuiltInFunctionCall(
 	return txProc.scProcessor.ExecuteBuiltInFunction(tx, acntSrc, acntDst)
 }
 
+func (txProc *txProcessor) processRelayedTxV2(
+	tx *transaction.Transaction,
+	relayerAcnt, acntDst state.UserAccountHandler,
+) (vmcommon.ReturnCode, error) {
+	if !txProc.flagRelayedTxV2.IsSet() {
+		return vmcommon.UserError, txProc.executingFailedTransaction(tx, relayerAcnt, process.ErrRelayedTxV2Disabled)
```

Comment:
to be fully backwards compatible we might execute the transaction as a move balance transaction

Reply:
that is resolved from the transactionType.go -> it returns the actual type. it is not only moveBalance, it can be sc invoking as well.

Reply:
ok